test/test_transforms.py

```diff
@@ -532,7 +532,13 @@ def test_convert_image_dtype_float_to_float(self):
             for output_dtype in output_dtypes:
                 with self.subTest(input_dtype=input_dtype, output_dtype=output_dtype):
                     transform = transforms.ConvertImageDtype(output_dtype)
+                    transform_script = lambda image: torch.jit.script(F.convert_image_dtype(image, output_dtype))  # noqa: E731
```

Comment:
@fmassa Are you OK with ignoring the assignment of a `lambda` here? IMO `def`ining this a proper function here clutters the test without adding much other than the linter is happy.